Subtester/ProjectSettings/AndroidResolverDependencies.xml

```diff
@@ -1,6 +1,6 @@
 <dependencies>
   <packages>
-    <package>com.revenuecat.purchases:purchases-hybrid-common:1.4.4</package>
```

Comment:
not sure why this wasn't `1.4.5` before, we might have missed that when doing the previous update

Reply:
I think it's because the project was never ran and it never got updated. This is like a `package.json.lock`, so if the project is never ran with the new dependencies, it won't get updated. That's my guess. Thanks for updating it!